src/components/main-view/main-view.jsx

```diff
@@ -1,48 +1,126 @@
 import React from 'react';
 import axios from 'axios';
 
-import { MovieCard } from '../movie-card/movie-card';
 import { MovieView } from '../movie-view/movie-view';
+import { MovieCard } from '../movie-card/movie-card';
+import { LoginView } from '../login-view/login-view';
+import { RegistrationView } from '../registration-view/registration-view';
```

Comment:
here is the import but you use other name.. you should define as 
import { RegisterView as  RegistrationView } from '../registration-view/registration-view';
or put the right name